iothub_client/src/iothub_client_core.c

```diff
@@ -2486,6 +2486,42 @@ IOTHUB_CLIENT_RESULT IoTHubClientCore_UploadMultipleBlocksToBlobAsync(IOTHUB_CLI
     return result;
 }
 
+IOTHUB_CLIENT_LL_UPLOADTOBLOB_CONTEXT_HANDLE IoTHubClientCore_CreateUploadContext(IOTHUB_CLIENT_CORE_HANDLE iotHubClientHandle, const char* destinationFileName)
+{
+    IOTHUB_CLIENT_LL_UPLOADTOBLOB_CONTEXT_HANDLE result;
+
+    if (
+        (iotHubClientHandle == NULL) ||
```

Comment:
Nit: same line?

Reply:
I went too far trying to keep the format used along the file. Fixed.